packages/visualizations/src/components/Table/Row.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+    import Cell from './Cell';
+    import ZoomIcon from './ZoomIcon.svelte';
+    import type { Column, Rows } from './types';
+
+    export let columns: Column[];
+    export let rows: Rows | undefined;
+    export let record: Record<string, unknown>;
+    let isRowHovered = false;
+
+    $: onMouseEnter = () => {
+        if (rows?.onMouseEnter) {
+            rows.onMouseEnter(record);
+        }
+        isRowHovered = true;
+    };
+
+    $: onMouseLeave = () => {
+        if (rows?.onMouseLeave) {
+            rows.onMouseLeave(record);
+        }
+        isRowHovered = false;
+    };
+
+    $: onClick = () => {
+        if (rows?.onClick) {
+            rows.onClick(record);
+        }
+    };
+</script>
+
+<tr on:mouseenter={onMouseEnter} on:mouseleave={onMouseLeave}>
+    {#if rows?.onClick}
+        <td class="button-cell">
+            <button on:click={onClick} class:visible={isRowHovered}>
```

Comment:
There's no text in the button so it probably needs an `aria-label`. I'm not sure there's something generic enough, maybe it's worth an exposed property (here we'd pass "Open record" I guess).